runtime/compiler/build/toolcfg/gnu/common.mk

```diff
@@ -98,13 +98,14 @@ C_FLAGS+=\
 
 CX_FLAGS+=\
     -pthread \
-    -fomit-frame-pointer \
     -fasynchronous-unwind-tables \
     -Wreturn-type \
     -fno-dollars-in-identifiers
 
+    #-fomit-frame-pointer \
```

Comment:
Is there a reason this was commented out?  Better to delete the code than to leave it commented out.